### ISSUE TITLE
Mumble 1.4.230 fails to launch after installation

### ISSUE THREAD

**Issue by basharoot** [mumble-voip/mumble#5465]
### The issue

Uninstalled 1.3.X from my computer and installed 1.4.230. Worked on one machine, but silently doesn't launch on the second machine after a restart. Attempts to freshly uninstall everything and reinstall did not help.
  
Checked the Event Log and got a very cryptic message:
>Faulting application name: mumble.exe, version: 1.4.230.0, time stamp: 0x61d1e7c3
Faulting module name: ntdll.dll, version: 10.0.19041.1466, time stamp: 0xe2f8ca76
Exception code: 0xc0000026
Fault offset: 0x0000000000102346
Faulting process id: 0x18d8
Faulting application start time: 0x01d80c8e0431ad84
Faulting application path: C:\Program Files\Mumble\client\mumble.exe
Faulting module path: C:\WINDOWS\SYSTEM32\ntdll.dll
Report Id: f673b447-c88e-4691-a921-116f690b28bb
Faulting package full name: 
Faulting package-relative application ID: 

Process Monitor has a more robust trace. Seems a big chunk is related to registry keys. Unsure if it is safe to attach to the Issue, but let me know if y'all would like the PML/XML/CSV dump of the logfile.
 
If there's anything else I can do to help chase this down, let me know. Appreciate the mountain of work y'all do!
Bash

### Mumble version

1.4.230

### Mumble component

Client

### OS

Windows

**Comment by Krzmbrzl** [comment]
Do you have the Visual C++ Redistributable installed on your second PC? Normally it should be installed (apparently it's shipped via Windows updates these days) but let's double-check that first :eyes:

**Comment by basharoot** [comment]
Holy smokes you must eat lightning for breakfast; thank you for the fast response!
 
I made sure to grab the msvc redist from [this page](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170).
I got an error on the first machine that led me to this. So I made sure the second machine had this before I did anything else.
  
For clarity: the machine that is working properly is the murmur server, though I also install the client on this machine for doing SuperUser tasks. I did run into a fun issue that is unrelated to this one- murmur.ini was missing (even the original). no matter how many installs or uninstalls I did. I just grabbed a copy from the github repo and slapped it in the %appdata% local path and got it running again. Haven't seen that happen in years prior but I digress from our main topic.
  
Back to the troublemaking machine: I went to double check and it seems I had both a 14.28 and a 14.30 version for msvc 2015-2022. Uninstalled the older one and redid the installation on the newer one. Completely uninstalled the mumble client, restarted, reinstalled. No luck, sadly. I can generate fresh logs from Process Monitor to see if anything improved.
 
Much obliged,
Bash

**Comment by Krzmbrzl** [comment]
Hm... what version of Windows are you using?

**Comment by basharoot** [comment]
Windows 10 Pro Version 20H2

**Comment by davidebeatrici** [comment]
I would remove the registry-related part from the dump, as sensitive info such as the certificate is stored there.

**Comment by Krzmbrzl** [comment]
From https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/596a1078-e883-4972-9bbc-49e60bebca55 the exception code `0xC0000026` that is mentioned in the log you posted means
> STATUS_INVALID_DISPOSITION: An invalid exception disposition was returned by an exception handler.

I don't know what that means though.

I think it might provide some insights if you could share as much of the dump as possible :eyes:

**Comment by basharoot** [comment]
> I would remove the registry-related part from the dump, as sensitive info such as the certificate is stored there.

@davidebeatrici I was able to filter out the registry related items from the Process Monitor logs, but it still seems pretty sensitive.
@Krzmbrzl Could I encrypt the logs and upload them to this Issue? I know the website says not to use email for general support, but I could use [onetimesecret](https://onetimesecret.com/) to email the decryption key to the security inbox. I am also willing to send the decryption key via private message on Matrix.

**Comment by Krzmbrzl** [comment]
>  I know the website says not to use email for general support,

That's mostly to prevent users from ignoring all public channels and sending an email instead. In this case it'd be just fine if you sent us the logs by mail. Send it to the contact email though.

**Comment by basharoot** [comment]
Sent! ✨📧✨

**Comment by davidebeatrici** [comment]
jackaudio/jack2#332

```xml
<event>
<ProcessIndex>450</ProcessIndex>
<Time_of_Day>10:58:12.3300764 AM</Time_of_Day>
<Process_Name>mumble.exe</Process_Name>
<PID>2688</PID>
<Operation>CreateFile</Operation>
<Path>C:\Windows\libjack64.dll</Path>
<Result>SUCCESS</Result>
<Detail>Desired Access: Read Data/List Directory, Execute/Traverse, Synchronize, Disposition: Open, Options: Synchronous IO Non-Alert, Non-Directory File, Attributes: n/a, ShareMode: Read, Delete, AllocationSize: n/a, OpenResult: Opened</Detail>
<stack>
<frame>
<depth>0</depth>
<address>0xfffff8014d30601b</address>
<path>C:\WINDOWS\System32\drivers\FLTMGR.SYS</path>
<location>FltDecodeParameters + 0x213b</location>
</frame>

<frame>
<depth>1</depth>
<address>0xfffff8014d305c55</address>
<path>C:\WINDOWS\System32\drivers\FLTMGR.SYS</path>
<location>FltDecodeParameters + 0x1d75</location>
</frame>

<frame>
<depth>2</depth>
<address>0xfffff8014d33c270</address>
<path>C:\WINDOWS\System32\drivers\FLTMGR.SYS</path>
<location>FltAddOpenReparseEntry + 0x560</location>
</frame>

<frame>
<depth>3</depth>
<address>0xfffff8014e68f825</address>
<path>C:\WINDOWS\system32\ntoskrnl.exe</path>
<location>IofCallDriver + 0x55</location>
</frame>

<frame>
<depth>4</depth>
<address>0xfffff8014e690e24</address>
<path>C:\WINDOWS\system32\ntoskrnl.exe</path>
<location>IoGetAttachedDevice + 0x194</location>
</frame>

<frame>
<depth>5</depth>
<address>0xfffff8014ea7726d</address>
<path>C:\WINDOWS\system32\ntoskrnl.exe</path>
<location>NtDeviceIoControlFile + 0x25ed</location>
</frame>

<frame>
<depth>6</depth>
<address>0xfffff8014e9f23ee</address>
<path>C:\WINDOWS\system32\ntoskrnl.exe</path>
<location>SeOpenObjectAuditAlarmWithTransaction + 0x58ce</location>
</frame>

<frame>
<depth>7</depth>
<address>0xfffff8014ea9499a</address>
<path>C:\WINDOWS\system32\ntoskrnl.exe</path>
<location>ObOpenObjectByNameEx + 0x1fa</location>
</frame>

<frame>
<depth>8</depth>
<address>0xfffff8014ea1608f</address>
<path>C:\WINDOWS\system32\ntoskrnl.exe</path>
<location>NtCreateFile + 0x49f</location>
</frame>

<frame>
<depth>9</depth>
<address>0xfffff8014ea15bd8</address>
<path>C:\WINDOWS\system32\ntoskrnl.exe</path>
<location>NtOpenFile + 0x58</location>
</frame>

<frame>
<depth>10</depth>
<address>0xfffff8014e808cb8</address>
<path>C:\WINDOWS\system32\ntoskrnl.exe</path>
<location>setjmpex + 0x7c88</location>
</frame>

<frame>
<depth>11</depth>
<address>0x7ffe8858d3d4</address>
<path>C:\Windows\System32\ntdll.dll</path>
<location>ZwOpenFile + 0x14</location>
</frame>

<frame>
<depth>12</depth>
<address>0x7ffe8854fbbc</address>
<path>C:\Windows\System32\ntdll.dll</path>
<location>RtlUnlockHeap + 0x178c</location>
</frame>

<frame>
<depth>13</depth>
<address>0x7ffe8854f6d0</address>
<path>C:\Windows\System32\ntdll.dll</path>
<location>RtlUnlockHeap + 0x12a0</location>
</frame>

<frame>
<depth>14</depth>
<address>0x7ffe8854eb90</address>
<path>C:\Windows\System32\ntdll.dll</path>
<location>RtlUnlockHeap + 0x760</location>
</frame>

<frame>
<depth>15</depth>
<address>0x7ffe8850fb53</address>
<path>C:\Windows\System32\ntdll.dll</path>
<location>RtlGetFullPathName_UstrEx + 0x22c3</location>
</frame>

<frame>
<depth>16</depth>
<address>0x7ffe885073e4</address>
<path>C:\Windows\System32\ntdll.dll</path>
<location>RtlDosPathNameToNtPathName_U + 0xd4</location>
</frame>

<frame>
<depth>17</depth>
<address>0x7ffe88506af4</address>
<path>C:\Windows\System32\ntdll.dll</path>
<location>LdrLoadDll + 0xe4</location>
</frame>

<frame>
<depth>18</depth>
<address>0x7ffe85c2ade2</address>
<path>C:\Windows\System32\KernelBase.dll</path>
<location>LoadLibraryExW + 0x162</location>
</frame>

<frame>
<depth>19</depth>
<address>0x7ffe1467b8df</address>
<path>C:\Program Files\Mumble\client\mumble_app.dll</path>
<location>MumbleMain + 0xd636af</location>
</frame>

<frame>
<depth>20</depth>
<address>0x7ffe145704c6</address>
<path>C:\Program Files\Mumble\client\mumble_app.dll</path>
<location>MumbleMain + 0xc58296</location>
</frame>

<frame>
<depth>21</depth>
<address>0x7ffe13746695</address>
<path>C:\Program Files\Mumble\client\mumble_app.dll</path>
<location>mumbleSelfDetection + 0x1a455</location>
</frame>

<frame>
<depth>22</depth>
<address>0x7ffe1374dc57</address>
<path>C:\Program Files\Mumble\client\mumble_app.dll</path>
<location>mumbleSelfDetection + 0x21a17</location>
</frame>

<frame>
<depth>23</depth>
<address>0x7ffe1391bcd5</address>
<path>C:\Program Files\Mumble\client\mumble_app.dll</path>
<location>MumbleMain + 0x3aa5</location>
</frame>

<frame>
<depth>24</depth>
<address>0x7ffe13918620</address>
<path>C:\Program Files\Mumble\client\mumble_app.dll</path>
<location>MumbleMain + 0x3f0</location>
</frame>

<frame>
<depth>25</depth>
<address>0x7ff61c561fac</address>
<path>C:\Program Files\Mumble\client\mumble.exe</path>
<location>mumbleSelfDetection + 0x41c</location>
</frame>

<frame>
<depth>26</depth>
<address>0x7ff61c562506</address>
<path>C:\Program Files\Mumble\client\mumble.exe</path>
<location>mumbleSelfDetection + 0x976</location>
</frame>

<frame>
<depth>27</depth>
<address>0x7ffe88407034</address>
<path>C:\Windows\System32\kernel32.dll</path>
<location>BaseThreadInitThunk + 0x14</location>
</frame>

<frame>
<depth>28</depth>
<address>0x7ffe88542651</address>
<path>C:\Windows\System32\ntdll.dll</path>
<location>RtlUserThreadStart + 0x21</location>
</frame>
</stack>
</event>
```

Updating JACK should fix the issue.

**Comment by basharoot** [comment]
Greetings! I uninstalled JACK1, JACK2, and JackTrip. I reinstalled the latest version of JACK and JackTrip. Mumble 1.4.230 now launches properly! Thank you for linking to the GH issue; this isn't even a Mumble problem to begin with but you all were able to assist me regardless. I am deeply obliged for your help and the time you committed to me. Looking forward to many more years with Mumble. I'll be closing the issue with this comment but I hope it may help others in the future.
